src/main/java/HomePageAmazon.java

```diff
@@ -0,0 +1,24 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class HomePageAmazon {
+
+    WebDriver driver;
+
+    public HomePageAmazon(WebDriver driver){
+        this.driver = driver;
+    }
+
+    public HomePageAmazon writeNintendo() {
```

Comment:
please create a generic function like this: `search(String keyword)` so that I can reuse it later with another keyword